examples/platforms/cc2538/radio.c

```diff
@@ -453,11 +453,12 @@ void readFrame(void)
     length = HWREG(RFCORE_SFR_RFDATA);
     otEXPECT(IEEE802154_MIN_LENGTH <= length && length <= IEEE802154_MAX_LENGTH);
 
-#if OPENTHREAD_ENABLE_RAW_LINK_API
-    // Timestamp
-    sReceiveFrame.mInfo.mRxInfo.mMsec = otPlatAlarmMilliGetNow();
-    sReceiveFrame.mInfo.mRxInfo.mUsec = 0; // Don't support microsecond timer for now.
-#endif
+    if (otPlatRadioGetPromiscuous(NULL))
```

Comment:
I think it would be better to pass the proper instance, rather than `NULL`.  In this case, `sInstance` already exists. However, I understand, in some cases below, `sInstance` does not. Can we update the platform code to maintain `sInstance`?

Reply:
fixed.